openvm/src/customize_exe.rs

```diff
@@ -61,17 +61,19 @@ impl From<powdr_autoprecompiles::constraint_optimizer::Error> for Error {
     }
 }
 
-pub fn customize<P: IntoOpenVm>(
+pub fn customize(
```

Comment:
We lose field genericity here because air extraction is not generic. Field genericity is not a priority though.